kbase-extension/static/kbase/js/widgets/function_input/kbStandaloneListselect.js

```diff
@@ -140,7 +140,7 @@
 
         render: function (index) {
             const renderer = rendererListselect[index];
-
+            let button_span, result_list;
```

Comment:
need to be declared at this level as the previous code took advantage of variable hoisting